Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoBinder.cs

```diff
@@ -49,12 +43,15 @@ public static class HandBinderAutoRigger {
             handBinder.boundHand.fingers[4].boundBones = AssignUnityBone(pinkyBones);
             handBinder.boundHand.wrist = AssignBoundBone(wrist);
             handBinder.boundHand.elbow = AssignBoundBone(Elbow);
+
             if(wrist != null && Elbow != null) {
```

Comment:
So this highlights one question I have - why are some of the fields/properties capitalised and others not? Are wrist and Elbow not equivalent in terms of accessibility here?